nbconvert/templates/rst.tpl

```diff
@@ -50,10 +50,16 @@
 
 {% block data_png %}
 .. image:: {{ output.metadata.filenames['image/png'] | urlencode }}
+{%- if 'image/png' in output.metadata and output.metadata['image/png'].get('width', '') %}
```

Comment:
It's cleaner to use the [`get_metadata` filter](https://github.com/jupyter/nbconvert/blob/master/nbconvert/filters/metadata.py#L3). You can find an example of its use for an analogous purpose in the [html template](https://github.com/jupyter/nbconvert/blob/master/nbconvert/templates/html/basic.tpl#L149).